data/campaigns/Liberty/scenarios/05_Hide_and_Seek.cfg

```diff
@@ -46,10 +46,10 @@
         side=2
         canrecruit=yes
         controller=ai
-        ai_algorithm=idle_ai
         user_team_name=_"Asheviere"
         team_name=bad_guys
         [ai]
+            ai_algorithm=idle_ai
```

Comment:
This does mean that this side will never do anything - I assume that's intentional?